internal/argtype.go

```diff
@@ -159,6 +159,9 @@ type ArgType struct {
 	// ShortNameTypeMap is the collection of Go style short names for types, mainly
 	// used for use with declaring a func receiver on a type.
 	ShortNameTypeMap map[string]string `arg:"-"`
+
+	// UseNullableProcParams toggle nullable types for stored procedure parameters.
+	UseNullableProcParams bool `arg:"--use-nullable-proc-params"`
```

Comment:
I would just call it `NullableProcParams`/`--nullable-proc-params`

Reply:
I wasn't sure about this one, I rename it then